src/middlewares.js

```diff
@@ -110,6 +110,11 @@ export function handleParseHeaders(req, res, next) {
   req.config = new Config(info.appId, mount);
   req.info = info;
 
+  const ip = req.ip;
```

Comment:
Hey @flovilmart , @montymxb was right. We can handle the LB case in LB with something like: 

```
const ip = req.headers['x-forwarded-for'] ||
    req.connection.remoteAddress || ... etc
``` 

But the question is, should we try to handle that on parse-server side or let user configure that by using `app.enable('trust proxy');` in the beginning? 

Let me know what you think.